### DISCUSSION TITLE
how can the seb client get the configurations from a .seb file if its encrypted ?

### DISCUSSION THREAD

**Discussion by emanh1** [SafeExamBrowser/seb-win-refactoring#907] [General]
how can the seb client get the configurations from a .seb file if its encrypted with an admin password, and without me entering it in?

**Comment by dbuechel** [comment]
The admin password is only required for editing the configuration in the configuration tool, whereas a settings password is required to decrypt (i.e. use) the configuration for use in SEB.

**Reply by emanh1** [reply]
is it possible to edit the configuration if i set passwordIsHash to true in DecryptDataWithPasswordForConfiguringClient?

**Reply by dbuechel** [reply]
I'm not quite sure I understand what you're referring to; where do you set these values?
